### ISSUE TITLE
how dese Timer-XL do  multivariate time series forecasting and covariate-informed time series forecasting

### ISSUE THREAD

**Issue by veelion** [thuml/Large-Time-Series-Model#40]
The Timer-XL's benchmark looks perfect! 

But there is only a notebook example ([quickstart_zero_shot.ipynb](https://github.com/thuml/Large-Time-Series-Model/blob/main/examples/quickstart_zero_shot.ipynb) for univariate forecasting.

How to do multivariate/covariate forecasting?  Any examples? Thanks.

Best Regards.

**Comment by WenWeiTHU** [comment]
Hi, currently we provide a univariate-pre-trained version of Timer for zero-shot forecasting (namely training under channel independence). If you'd like to use the model for zero-shot forecasting, you can simply get the multivariate outputs by prediction channel by channel. If you are interested in supervised training (the performance would be better when training samples are available), you can start the training with the scripts in this [repo](https://github.com/thuml/OpenLTM/tree/main/scripts/supervised/rolling_forecast).

**Comment by veelion** [comment]
Thanks. Any plan for covariate-informed time series forecasting?

**Comment by WenWeiTHU** [comment]
We are currently occupied with improving the zero-shot performance (basically univariate). But I think covariate-informed is a very practical and valuable case :)

**Comment by veelion** [comment]
Yes, I completely agree with your point that covariate-informed prediction is very valuable, and I look forward to seeing your work on this come to fruition soon.
